Blueprints/BlueprintDefinitions/vs2022/Templates.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <PackageType>Template</PackageType>
-    <PackageVersion>7.3.0</PackageVersion>
+    <PackageVersion>7.4.0</PackageVersion>
```

Comment:
Unrelated to your change, but we should update the `autover` config file to automatically update the blueprints as well now that we no longer use `nuspec` files.